.github/workflows/tests runner.yml

```diff
@@ -23,7 +23,6 @@ jobs:
       - name: Check linting errors
         run: npm run lint
       - name: Run tests and generate coverage reports
```

Comment:
@Ke-bean , why is this run removed

Reply:
> @Ke-bean , why is this run removed

Was testing something and forget to bring back it, now it's fixed @aimedivin